src/components/press-custom-map-marker/CustomOverlay.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * factory for CustomOverlays
+ *
+ * @export
+ * @param {*} google
+ * @returns {*}
+ */
+export default function createCustomOverlay(google) {
+  /** custom overlay extending OverlayView */
+  return class CustomOverlay extends google.maps.OverlayView {
+    /**
+     * Creates an instance of CustomOverlay.
+     * @param {google.maps.LatLnglocation} location
+     * @param {HTMLElement} node
+     * @param {HTMLElement} container
+     * @param {google.maps.Map} map
+     */
+    constructor(location, node, container, map) {
+      super();
+      this.location = location;
+      this.node = node;
+      this.map = map;
+      this.container = container;
+      this.container.style.position = 'absolute';
+
+      this.setMap(map);
+    }
+    /**
+     * lifecycle method for OverlayView
+     *
+     * @memberof CustomOverlay
+     */
+    onAdd() {
+      this.container.append(this.node);
+
+      const panes = this.getPanes();
+      panes.markerLayer.appendChild(this.container);
+    }
+    /**
+     * lifecycle method for OverlayView
+     *
+     * @memberof CustomOverlay
+     */
+    draw() {
+      const overlayProjection = this.getProjection();
+
+      const sw = overlayProjection.fromLatLngToDivPixel(
+        this.location.getSouthWest()
+      );
+      const ne = overlayProjection.fromLatLngToDivPixel(
+        this.location.getNorthEast()
+      );
+
+      this.container.style.left = `${sw.x}px`;
+      this.container.style.top = `${ne.y -
```

Comment:
is `children[0]` guaranteed to exist here? 

```suggestion
      const {clientHeight} = this.container.children[0];
      this.container.style.top = `${ne.y - clientHeight/2}px`
```